### PR TITLE
Ember upgrade v4.4

### PR DESCRIPTION
**Pull request by benjay10**
Another upgrade for Ember (to version 4.4 LTS). Not much was needed for this.

### CODE REVIEW

**Comment by benjay10** [comment]
Did a quick check after the merge of the PR about the rewrite, and everything seems to work and look as expected.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: CI
```
Comment (Windvis) [inline .github/workflows/ci.yml:1]: This file probably needs to be removed (or replaced with Woodpecker / Drone in a different PR)?

Comment (benjay10) [reply .github/workflows/ci.yml:1]: Oi I didn't realise Ember generated this file during the upgrade. I don't think we use a set-up like that on this repo, the hosting is done through Github itself, via the gh-pages branch.

Comment (Windvis) [reply .github/workflows/ci.yml:1]: It could still be useful for running the linters.

**Review by Windvis — APPROVED** [review]

**Pull request merged**